src/data.js

```diff
@@ -43,6 +45,21 @@ const getReleasesOrUpdate = pMemoize(
   },
 );
 
+const getDownloadStatsOrUpdate = pMemoize(
+  async () => {
+    const response = await fetch('https://electron-sudowoodo.herokuapp.com/release/active');
+    return response.json();
+    return {
+      electron: electronJSON,
+      nightly: nightlyJSON,
+    };
+  },
+  {
+    cache: new ExpiryMap(60 * 1000),
+    cacheKey: () => 'download_stats',
+  },
+);
+
```

Comment:
This looks unused?

Reply:
@dsanders11 correct, leftover from another PR i was working on 😅